data/transport missions.txt

```diff
@@ -1951,3 +1951,381 @@ ship "Science Drone"
 		"heat dissipation" 1
 	explode "tiny explosion" 5
 	explode "small explosion" 2
+
+
+
+mission "Lost Boy 1"
+	name `Rescue Tod`
```

Comment:
I've not seen `Tod`, usually `Todd`

Reply:
https://en.wikipedia.org/wiki/The_Fox_and_the_Hound

Reply:
Oh I checked on the name... Tod's just a lot rarer than Todd, and Todd is already pretty rare:
Tod:
![image](https://user-images.githubusercontent.com/20871346/31864422-b9e3febe-b722-11e7-977d-d545e66fc5c2.png)
Todd:
![image](https://user-images.githubusercontent.com/20871346/31864425-ce5d9058-b722-11e7-8674-0164d2a3e938.png)

So people's first instinct may be to think "typo" and not "Hi, Tod"

Reply:
Given that it's suppose to be a reference to The Fox and the Hound, I'd rather not change it.